jest.config.json

```diff
@@ -12,12 +12,7 @@
     "jest-canvas-mock"
   ],
   "snapshotSerializers": ["enzyme-to-json/serializer"],
-  "testMatch": [
-    "**/__spec__.js",
-    "**/*.spec.js",
-    "**/*.spec.ts",
-    "**/*.spec.tsx"
-  ],
+  "testMatch": ["**/?(*.)+(spec|test).[jt]s?(x)"],
```

Comment:
Added to match naming proposed naming pattern for RTL tests once linting is integrated